AGPs/AGP-Comprehensive-Aragon-Bug-Bounty-Program.md

```diff
@@ -0,0 +1,48 @@
+---
+AGP: N/A
+Title: Comprehensive Aragon Bug Bounty Program
+Author: burrrata (@burrrata)
+Status: Stage IV
+Track: Association
+Created: 2019-10-02
+---
+
+# AGP-X: Comprehensive Aragon Bug Bounty Program
+
+## Description of desired Association policy change
+
+This proposal is to expand the [Aragon Association Bug Bounty Program](https://wiki.aragon.org/association/security/bug_bounty/) to make all Aragon projects funded by the AA via AGPs, the Nest program, or the Flock program eligible. This definition of "funded by the AA" is left broad enough to cover a range of projects from the Aragon Court to Aragon app developed through the Nest program. This does not automatically include all such project, but makes them eligible for the bug bounty program. Projects will only be accepted into the bug bounty program if they have undergone a security audit, incorporated all major and critical security audit recommendations, and shipped to mainnet. If projects make major changes after incorporating security audits and shipping to mainnet, a new security audit will be required for the projects continued eligibility in the bug bounty program. This ensures that only projects that have been audited and incorporated critical and major security audit recommendations are included. Failure to undergo a security audit or to incorporate a security team's critical and major recommendations will result in projects being dropped from the Aragon Association Bug Bounty Program.
+
+In addition to expanding the scope of the [Aragon Association Bug Bounty Program](https://wiki.aragon.org/association/security/bug_bounty/), the amounts will also be changed.
+
+Currently the Aragon Association Bug Bounty program is as follows:
+- Critical (CVSS 9.0 - 10.0): $5,000 - $50,000
+- Major (CVSS 7.0 - 8.9): $2,500 - $5,000
+- Medium (CVSS 4.0 - 6.0): $1,000 - $2,500
+- Low (CVSS 1.0 - 3.9): $500 - $1,000
+
+The Aragon Association Bug Bounty program would change to the following:
+- Critical (CVSS 9.0 - 10.0): $50,000 - $150,000
+- Major (CVSS 7.0 - 8.9): $25,000 - $50,000
+- Medium (CVSS 4.0 - 6.0): $10,000 - $25,000
+- Low (CVSS 1.0 - 3.9): $5000 - $10,000
+
+A cap of $1,000,000 will be set for the program. This will allow Aragon to provide meaningful rewards for bug bounties while also limiting total liabilities for the Araogn Association. In the event that total liability of the Aragon Association Bug Bounty program exceeds $1,000,000 the Aragon Association will select which projects are covered. This selection will be based on the usage and utility of projects covered by the program.
```

Comment:
```suggestion
A cap of $1,000,000 will be set for the program. This will allow Aragon to provide meaningful rewards for bug bounties while also limiting total liabilities for the Aragon Association. In the event that total liability of the Aragon Association Bug Bounty program exceeds $1,000,000 the Aragon Association will select which projects are covered. This selection will be based on the usage and utility of projects covered by the program.
```